lib/watcher/status-wd.js

```diff
@@ -7,6 +7,7 @@ exports.worker = function(handle) {
       pwd: process.env.PWD,
       cwd: process.cwd(),
       pid: process.pid,
+      isTracing: !!process.env.STRONGLOOP_TRACING,
```

Comment:
this won't work. you'll have to rebase on top of a PR I'm about to make

won't work, because the env var only reflects desired state, the tracing might not actually be running.

actually, I think you can change this right now to

```
var tracer = require('../tracer');
...
isTracing: !!tracer(),
```